contrib/nydus-backend-proxy/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "nydus-backend-proxy"
+version = "0.1.0"
+authors = ["The Nydus Developers"]
+edition = "2018"
+license = "Apache-2.0 OR BSD-3-Clause"
+
+[dependencies]
+rocket = "0.5.0-rc"
```

Comment:
Can we use a stable version?

Reply:
> Can we use a stable version?

I get some errors(https://github.com/SergioBenitez/Rocket/issues/1908) when using Rocket v0.4.10, and it seems can't be resolved in a 0.4.x release. So can I update it after to a stable version of v0.5 ?

Reply:
Ok